### ISSUE TITLE
Include "$" in front of available shipping options

### ISSUE THREAD

**Issue by jordanmkoncz** [silvershop/silvershop-shipping#31]
Shipping options currently display just "`Price` - `Title` - `Description`", e.g. "10.00 - Standard Shipping - Description". To avoid any potential confusion, it should include a dollar sign in front of the price, e.g. "$10.00 - Standard Shipping - Description".

**Comment by imagicgit** [comment]
Workaround is to add CSS 

`#Form_ShippingMethodForm_ShippingMethodID {
 li {
     label {
         &:before {
             content: '$'
         }
     }
}`

**Comment by TheBnl** [comment]
I've made a PR #37 that adds an extension hook to the getTitle method. This gives you the ability to alter the title in an extension. You could add the currency like:
```php
public function updateTitle(&$title)
{
    $currency = Config::inst()->get('ShopCurrency', 'currency_symbol');
    $title = "{$currency}{$title}";
}
```

**Comment by wilr** [comment]
Fixed with #37.
